contrib/README.md

```diff
@@ -294,9 +294,11 @@ As long as either the native `Promise` is there, or that global has been spec-co
 
 ### `errfcb` Plugin
 
-`errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` returns an "error-first" style (aka "node-style") callback that can be used with any function that expects such a callback.
+The `errfcb` plugin provides `errfcb()` on the main sequence instance API. Calling `errfcb()` returns an "error-first" style (aka "node-style") callback that can be used with any function that expects such a callback.  Calling `errfcb()` also stops the sequence from proceeding until the "error-first" callback is invoked.
 
-If the "error-first" callback is then invoked with the first ("error") parameter set, the main sequence is flagged for error as usual. Otherwise, the main sequence proceeds as success. Messages sent to the callback are passed through to the main sequence as success/error as expected.
+If the "error-first" callback is then invoked with the first ("error") parameter having a truthy value, the main sequence is flagged for error as usual.  The value of the "error" parameter is provided to `.or()` callbacks as the failure reason.
+
+If the "error" parameter has a falsy value, the main sequence proceeds as success.  Any other values provided to the callback are passed through to the main sequence and can be accessed by later steps.
```

Comment:
Let's reword this just a bit for more clarity:

----

Any other values provided to the callback are passed through as normal messages to the main sequence, and can be accessed by the next step in the sequence.